frontend/kesaseteli/handler/src/pages/create-application-without-ssn.tsx

```diff
@@ -0,0 +1,52 @@
+import CreateApplicationWithoutSsnForm from 'kesaseteli/handler/components/form/CreateApplicationWithoutSsnForm';
+import type { ApplicationWithoutSsnFormData } from 'kesaseteli/handler/types/application-without-ssn-types';
+import { GetStaticProps, NextPage } from 'next';
+import Head from 'next/head';
+import { useTranslation } from 'next-i18next';
+import React from 'react';
+import { FormProvider, useForm } from 'react-hook-form';
+import Container from 'shared/components/container/Container';
+import FormSection from 'shared/components/forms/section/FormSection';
+import FormSectionHeading from 'shared/components/forms/section/FormSectionHeading';
+import getServerSideTranslations from 'shared/i18n/get-server-side-translations';
+
+const createApplicationWithoutSsn: NextPage = () => {
+  // eslint-disable-next-line react-hooks/rules-of-hooks
```

Comment:
These rules-of-hook linter disablings should not be needed here. Is it because the component name is as lower camel case?

Reply:
If the `// eslint-disable-next-line react-hooks/rules-of-hooks` line is removed then "yarn lint" will fail:

> yjdh\frontend\kesaseteli\handler\src\pages\create-application-without-ssn.tsx
  14:17  error  React Hook "useTranslation" is called in function "createApplicationWithoutSsn: NextPage" that is neither a React function component nor a custom React Hook function. React component names must start with an uppercase letter. React Hook names must start with the word "use"  react-hooks/rules-of-hooks
>
> ✖ 1 problem (1 error, 0 warnings)
>
> error Command failed with exit code 1.

Reply:
Renamed `createApplicationWithoutSsn` to `CreateApplicationWithoutSsn` and removed all eslint disablings from the file, "yarn lint" passes.

Reply:
Taken into account in [force push](https://github.com/City-of-Helsinki/yjdh/compare/9ce2658ba09ecbf6374e149c9d2ca5efc00b9c71..52f0669b62b54509a69ebbcdab31cb4aca8267f6)